lib/xfstests_utils.pm

```diff
@@ -20,7 +20,7 @@ use testapi;
 use utils;
 use Utils::Backends 'is_pvm';
 use serial_terminal 'select_serial_terminal';
-use power_action_utils qw(power_action prepare_system_shutdown);
```

Comment:
Maybe keep `power_action`? In your new line 630, there is `power_action` used.

Reply:
Fixed, thanks for reviewing!